### ISSUE TITLE
Undead Spawning during the day

### ISSUE THREAD

**Issue by GrundyT** [EdenServer/community#125]
I was in Valkrum Dunes and we had Ghouls remaining out after night was over and spawning during the day.

**Comment by Siushaga** [comment]
Bump, still got undead up at 10:00 AM in the Dunes. Haven't checked other zones.

**Comment by rujind** [comment]
Yeah something is throwing off the undead/bat timer, it's happed a few times this week. Undead/bats are up during the day and not spawning at night.

**Comment by Eydaos** [comment]
This is happening in Konschtat Highlands, too.

**Comment by Frutiebutie** [comment]
in KRT as well, it is preventing Spook for spawning as well.
